### ISSUE TITLE
Unable to set output archive name

### ISSUE THREAD

**Issue by nslingerland** [nebula-plugins/gradle-ospackage-plugin#338]
The `archiveName` parameter works in the 5.3.5 (and earlier) versions of ospackage - but not in 6.x:

```
task distLinuxRpm(type: Rpm) {
    version "${package_version}"
    release "${package_release}"
    archiveName "${project.name}-${package_version}-${package_release}.linux.rpm"
    os LINUX
}
```
With 6.x `archiveName` seems to be ignored.

**Comment by elusiveeagle** [comment]
Yup, we noticed this as well. I understand that we can use the properties in the `Rpm` task to set values that will make it into the archive filename. However, in our case we add the release year and quarter to _just_ the filename. Said differently, we need the RPM to have a semantic version (X.Y.Z) but the filename needs to have the additional details. Here's an example of how it worked for us previously:

Package version: `1.2.3`
Package filename: `name-2019Q3.1.2.3-1.x86_64.rpm`

I tried all sorts of variations of the `archiveName`, `archiveFileName`, and `archiveFile` properties. Then I dug into the plugin code and it seems the behavior change we are seeing is caused by a combination of [Gradle 5.1 changing `AbstractArchiveTask` to use the Provider API](https://github.com/gradle/gradle/pull/7435) and this [plugin's corresponding changes to support Gradle 5.1](https://github.com/nebula-plugins/gradle-ospackage-plugin/pull/311) (in particular commit https://github.com/nebula-plugins/gradle-ospackage-plugin/commit/6f2818aaea7962cead6b78790fc1ac67e0c09aaa).

@nslingerland What version of Gradle did you use when you tested version 5.3.5 and 6.x of this plugin?

**Comment by nslingerland** [comment]
We are using `gradlew` which reports:
```
./gradlew --version

------------------------------------------------------------
Gradle 5.2.1
------------------------------------------------------------

Build time:   2019-02-08 19:00:10 UTC
Revision:     f02764e074c32ee8851a4e1877dd1fea8ffb7183

Kotlin DSL:   1.1.3
Kotlin:       1.3.20
Groovy:       2.5.4
Ant:          Apache Ant(TM) version 1.9.13 compiled on July 10 2018
JVM:          1.8.0_191 (Oracle Corporation 25.191-b12)
OS:           Linux 4.16.18-178_fbk18_4909_g135ce1e5547d amd64
```

**Comment by elusiveeagle** [comment]
@rpalcolea Howdy! I noticed you had tagged this with `Hacktoberfest`. Were you able to make any progress in October? Thanks.
